packages/react-app/src/views/BuilderProfileView.jsx

```diff
@@ -137,6 +205,27 @@ export default function BuilderProfileView({ serverUrl, mainnetProvider, address
                 </Box>
               </Flex>
             ))}
+          {!isLoadingBuilder && withdrawEvents.length && (
```

Comment:
For some reason this displays a "0" if empty.

See the bottom left of the image

![image](https://user-images.githubusercontent.com/2486142/157724870-575f3277-aedd-4556-af71-d020d122c8fe.png)

Reply:
This is an interesting one to keep in mind when doing boolean coercion from falsy but not false values

```jsx
{!isLoadingBuilder && withdrawEvents.length && <SomeComponent />}

// vs 

{!isLoadingBuilder && withdrawEvents.length !== 0 && <SomeComponent />}
```

The first one checks the first statement, is a condition that is met, continues. Checks the second statement, it's a number, which is a valid return from a js block in JSX, returns the number.

The second one does the same, but the second statement is a condition, so it checks if it's true to continue, or false to avoid processing the 3rd statement.